### PR TITLE
Add security contacts metadata

### PR DESCRIPTION
**Pull request by daniel-beck**
So far we tracked this internally, but it cannot hurt adding it to the metadata here, and giving maintainers an easy way to define this themselves.

### CODE REVIEW

File: permissions/plugin-github.yml [hunk 1]
```diff
@@ -7,3 +7,6 @@ developers:
 - "integer"
 - "lanwen"
 - "oleg_nenashev"
+security:
+  contacts:
+    jira: "integer"
```
Comment (daniel-beck) [inline permissions/plugin-github.yml]: FYI @KostyaSha

**Comment by daniel-beck** [comment]
@halkeye Could you review this to see whether I included everything in your security group's domain?

File: permissions/plugin-maven-plugin.yml [hunk 2]
```diff
@@ -11,3 +11,6 @@ developers:
 - "olamy"
 - "olivergondza"
 - "aheritier"
+security:
```
Comment (daniel-beck) [inline permissions/plugin-maven-plugin.yml]: @olamy @aheritier FYI

**Comment by halkeye** [comment]
> @halkeye Could you review this to see whether I included everything in your security group's domain?

I think that's good. Nothing stands out as missing. Officially I think we also maintain the jira-plugin (not the blueocean integration) but i don't think that makes sense.

**Pull request merged**